client/src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Icons from '../Icons/FAIcon';
+import './NavBar.css';
+import logo from './Logo/Logo.png';
```

Comment:
There's a logo component that you could use insteadhere

Reply:
Lookslike you missed this one?

Reply:
Added a placeholder till the logo component is complete